pkg/controller/jobs/raycluster/raycluster_controller_test.go

```diff
@@ -328,7 +328,6 @@ func TestReconciler(t *testing.T) {
 					Condition(metav1.Condition{
 						Type:               kueue.WorkloadRequeued,
 						Status:             metav1.ConditionTrue,
-						Reason:             "Pending",
```

Comment:
Why is this reason empty?

Reply:
On the request we are using abstract (without Reason and Message) evicted condition. So thats why on result we get it empty. I will set `WorkloadEvictedByDeactivation`.

Reply:
Done